packages/core/lib/config.js

```diff
@@ -90,12 +90,17 @@ const loadConfig = (context, config) => {
     : searchSync(context);
 };
 
-const loadSettings = (context) => {
+const loadSettings = (context, { dependencies }) => {
   const result = loadConfig(context);
-  return {
-    presets: ['@untool/defaults'],
+  const settings = {
     ...(result ? result.config : {}),
   };
+  if (!settings.presets) {
+    settings.presets = Object.keys(dependencies).filter((key) =>
```

Comment:
I would prefer to pass in the dependencies as an array in the first place.

Reply:
Sorry, was a bit too quick: https://github.com/untool/untool/releases/tag/v0.12.0